frontend/components/transaction/TransactionFeesTable.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+    <div v-if="isWalletConnected" class="flex flex-col space-y-1 text-gray-700 dark:text-gray-100">
+        <div v-if="swapTransactionFeeETH" class="flex justify-between">
+            <div>Swap Transaction Fee</div>
+            <div>
+                <FormatCurrency :value="swapTransactionFeeETH" :decimals="5" :currency="ETH" />
```

Comment:
For some reason, neither on the screenshots nor in my testing can I see the currency symbol behind the number. Please ensure it says `ETH` behind the numbers so the users know what currency it is in. 

@LukSteib maybe we also want to display the DAI amount for them? As 1 DAI = 1 USD, it may give the user better insight then the ETH amount.